### ISSUE TITLE
cond_mean, cond_std uses too many dims for vector attr

### ISSUE THREAD

**Issue by v0xie** [v0xie/sd-webui-semantic-guidance#10]
`cond_mean, cond_std = torch.mean(concept_cond, dim=(-3,-2,-1)), torch.std(concept_cond, dim=(-3,-2,-1))`
When dim of concept_cond is 3 it will use 1 more dim than needed
